src/main/java/org/myrobotlab/inmoov/Vision.java

```diff
@@ -110,4 +132,47 @@ public boolean isTracking() {
   public boolean isReady() {
     return ready;
   }
+
+  /**
+   * sub onClassification to a analyze yolo filter, and save some code lines from main service
+   */
+  public void onClassification(TreeMap<String, List<Classification>> classifications) {

```

Comment:
Seems like the wrong place for this method.  Typically only services should have things invoked and published on them.  Vision is not a service.

Reply:
Let's keep callback/subscribe methods to Service classes if we can.